proc_scripts/load_data.py

```diff
@@ -45,13 +45,15 @@ def proc_CPMG(s):
     s.set_units('t','s')
     twice_tau = deblank_s + 2*p90_s + deadtime_s + pad_start_s + acq_time_s + pad_end_s + marker_s
     t2_axis = linspace(0,acq_time_s,nPoints)
-    tE_axis = r_[1:nEchoes+1]*twice_tau
-    s.setaxis('t',None)
     s.setaxis('nScans',r_[0:nScans])
```

Comment:
in general, this business where you are setting the time axis to `None` is never the most recent version of the code.  If you just apply chunk first to a labeled axis, it will unspool the axis labels in a way that is sensible.  (See the manpage for chunk, and pull request if needed).  Note that we only have to relabel the echo axis, because there are additional "pauses" where we close the receiver and execute a pulse, and the pauses are not included in the original time axis.

Reply:
(setting the time axis to `None` is an artifact of older versions of pyspecdata, which couldn't handle relabeling the axis during a chunk)

Reply:
so, just double-checking that the CPMG code works with these changes?

Reply:
yes